### PR TITLE
Ticket7024 dae make apply changes more obvious

### PR DESCRIPTION
**Pull request by ThomasLohnert**
### Description of work

- Made the "Apply Changes" button much bigger on the DAE Experiment Setup tab
- Added an indicator for unapplied changes to the DAE run summary tab (front panel).

### Ticket

https://github.com/ISISComputingGroup/IBEX/issues/7024

### Acceptance criteria

See ticket

### Unit tests

/

### System tests

/

### Documentation

/

---

#### Code Review

- [ ] Is the code of an acceptable quality?
- [ ] Do the changes function as described and is it robust?
- [ ] Is there associated PR for the [release notes](https://github.com/ISISComputingGroup/IBEX/blob/master/release_notes/ReleaseNotes_Upcoming.md)?

### Final Steps
- [ ] Reviewer has also merged the [release notes](https://github.com/ISISComputingGroup/IBEX/blob/master/release_notes/ReleaseNotes_Upcoming.md)

### CODE REVIEW

File: base/uk.ac.stfc.isis.ibex.ui.dae/src/uk/ac/stfc/isis/ibex/ui/dae/run/RunSummary.java [hunk 1]
```diff
@@ -121,8 +125,13 @@ public void createPart(Composite parent) {
 		runStatus.setLayoutData(gdRunStatus);
 		runStatus.setText("UNKNOWN");
 
-		Label spacer = new Label(infoComposite, SWT.NONE);
-		spacer.setLayoutData(new GridData(SWT.LEFT, SWT.CENTER, true, false, 1, 1));
+		changeIndicator = new Label(infoComposite, SWT.NONE);
+		changeIndicator.setLayoutData(new GridData(SWT.RIGHT, SWT.CENTER, true, false, 1, 1));
+        String defaultFont = changeIndicator.getFont().getFontData()[0].getName();
+        Font biggerFont = new Font(changeIndicator.getDisplay(), new FontData(defaultFont, 11, SWT.BOLD));
+        changeIndicator.setFont(biggerFont);
```
Comment (rerpha) [inline base/uk.ac.stfc.isis.ibex.ui.dae/src/uk/ac/stfc/isis/ibex/ui/dae/run/RunSummary.java:132]: looks like there's some funky indentation going on here

File: base/uk.ac.stfc.isis.ibex.ui.dae/src/uk/ac/stfc/isis/ibex/ui/dae/run/RunSummary.java [hunk 2]
```diff
@@ -180,8 +189,15 @@ public void widgetSelected(SelectionEvent e) {
 		daeButtonPanel.setLayoutData(new GridData(SWT.FILL, SWT.FILL, false, false, 1, 1));
 
         setModel(model);
+        bindChangeIndicator();
 	}
 
+    private void bindChangeIndicator() {
+		DataBindingContext bindingContext = new DataBindingContext();
+        ExperimentSetupViewModel experimentSetupModel = DaeUI.getDefault().viewModel().experimentSetup();
```
Comment (rerpha) [inline base/uk.ac.stfc.isis.ibex.ui.dae/src/uk/ac/stfc/isis/ibex/ui/dae/run/RunSummary.java:197]: Maybe here too

**Comment by Adam-Szw** [comment]
Looks good now after formatting @ThomasLohnert . Tested features too. Let me know if that's it for the commits and I will see again and merge.

**Comment by ThomasLohnert** [comment]
@Adam-Szw yup, all done as far as I'm concerned, cheers

**Review by Adam-Szw — APPROVED** [review]

**Pull request merged**